ios/FluentUI/Tab Bar/TabBarItemView.swift

```diff
@@ -271,16 +264,20 @@ class TabBarItemView: UIControl {
     }
 
     private func updateColors() {
-        let primaryColor = UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.brandForeground1])
-        titleLabel.highlightedTextColor = primaryColor
-        imageView.tintColor = isSelected ? primaryColor : unselectedColor
+        let selectedColor = UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.brandForeground1])
+        let unselectedImageColor = UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.foreground3])
+        let unselectedTextColor = UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.foreground2])
+        let disabledColor = UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.foregroundDisabled1])
+
+        titleLabel.textColor = isEnabled ? (isSelected ? selectedColor : unselectedTextColor) : disabledColor
+        imageView.tintColor = isEnabled ? (isSelected ? selectedColor : unselectedImageColor) : disabledColor
     }
 
     private func updateImage() {
         // Normally we'd set imageView.image and imageView.highlightedImage separately. However, there's a known issue with
```

Comment:
is the workaround fix that @mischreiber did for iOS16 on boot still working?

Reply:
My understanding is this doesn't affect the fact that we avoid using `highlightedImage`. We now just directly use the `isSelected` state of the whole item, which previously translated to the image's `isHighlighted` anyways, so 283 has the same behavior as before.